src/patternfly/components/InputGroup/input-group.scss

```diff
@@ -1,40 +1,41 @@
 // @debug $input-group; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
+:root,
 .#{$input-group} {
-  // Input group
-  --#{$input-group}--child--ZIndex: var(--#{$pf-global}--ZIndex--xs);
+  --#{$input-group}--Gap: var(--pf-t--global--spacer--xs);
 
   // Input group item
-  --#{$input-group}__item--offset: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$input-group}__item--MarginLeft: calc(var(--#{$input-group}__item--offset) * -1);
-  --#{$input-group}__item--BorderWidth--base: var(--#{$input-group}__item--offset);
-  --#{$input-group}__item--BorderColor--base: var(--#{$pf-global}--BorderColor--300);
-  --#{$input-group}__item--BorderColor--accent: var(--#{$pf-global}--BorderColor--200);
+  --#{$input-group}__item--offset: var(--pf-t--global--border--width--control--default);
+  --#{$input-group}__item--BorderWidth--base: var(--pf-t--global--border--width--control--default);
+  --#{$input-group}__item--BorderColor--base: var(--pf-t--global--border--color--default);
   --#{$input-group}__item--BackgroundColor: transparent;
 
   // Input group item, box variant
-  --#{$input-group}__item--m-box--PaddingRight: var(--#{$pf-global}--spacer--sm);
-  --#{$input-group}__item--m-box--PaddingLeft: var(--#{$pf-global}--spacer--sm);
-  --#{$input-group}__item--m-box--BackgroundColor: var(--#{$pf-global}--BackgroundColor--100);
-  --#{$input-group}__item--m-box--BorderWidth: var(--#{$input-group}__item--offset);
+  --#{$input-group}__item--m-box--PaddingRight: var(--pf-t--global--spacer--sm);
+  --#{$input-group}__item--m-box--PaddingLeft: var(--pf-t--global--spacer--sm);
+  --#{$input-group}__item--m-box--BackgroundColor: var(--pf-t--global--background--color--secondary--default);
+  --#{$input-group}__item--m-box--BorderWidth: var(--#{$input-group}__item--BorderWidth--base);
+  --#{$input-group}__item--m-box--BorderRadius: var(--pf-t--global--border--radius--small);
   --#{$input-group}__item--m-box--BorderTopColor: var(--#{$input-group}__item--BorderColor--base);
   --#{$input-group}__item--m-box--BorderRightColor: var(--#{$input-group}__item--BorderColor--base);
-  --#{$input-group}__item--m-box--BorderBottomColor: var(--#{$input-group}__item--BorderColor--accent);
+  --#{$input-group}__item--m-box--BorderBottomColor: var(--#{$input-group}__item--BorderColor--base);
   --#{$input-group}__item--m-box--BorderLeftColor: var(--#{$input-group}__item--BorderColor--base);
```

Comment:
Looks like this is broken. Should we remove the variant?

<img width="436" alt="Screenshot 2024-01-08 at 4 38 31 PM" src="https://github.com/patternfly/patternfly/assets/5385435/8e819ab3-726d-4550-afb1-adb767524210">

Reply:
That's used to put the extra things around it, like the $ or "@example.com" 
I think we still want that?